### ISSUE TITLE
Openchat3.5 training data

### ISSUE THREAD

**Issue by jzhang38** [imoneoi/openchat#60]
Congrats for the V3.5 release! 
May I ask if there are plans to release your finetuning data, just like what you have been always doing with your previous release?

**Comment by imoneoi** [comment]
Thanks! Sorry the dataset contains a bit private data, but we are considering releasing the open subsets of the dataset.

**Comment by yucc-leon** [comment]
Similar question. Did V3.5 still use the same strategy as the paper _openchat_ describe? Or there were other changes and you will publish a new-version report?

**Comment by imoneoi** [comment]
> Similar question. Did V3.5 still use the same strategy as the paper _openchat_ describe? Or there were other changes and you will publish a new-version report?

Yes, it's mainly based on the method that the paper described (except with more data). We may publish a new report if new modifications are made.

**Comment by timothylimyl** [comment]
@imoneoi if all data is generated using openai api, where does the private data come from?

**Comment by mietekrmd** [comment]
> @imoneoi if all data is generated using openai api, where does the private data come from?

I quess from books

**Comment by timothylimyl** [comment]
does your dataset have system prompts?
